lib/r10k/cli/deploy.rb

```diff
@@ -63,7 +63,8 @@ def self.command
 scheduled. On subsequent deployments, Puppetfile deployment will default to off.
           DESCRIPTION
 
-          flag :p, :puppetfile, 'Deploy modules from a puppetfile'
+          flag :p, :puppetfile, 'Deploy modules from a puppetfile (deprecated, use -m)'
```

Comment:
Should we change this description to also just say 'Deploy modules'? Otherwise, this suggests to me that the two flags have different behavior.